[15.0][MIG] commission, account_commission, sale_commission : Migration to 15.0 (#374)
OCA/commission: account_commission/readme/CONFIGURE.rst

For selecting invoice status in commissions:

#. Go to *Invoicing > Configuration > Commission Management > Commission types*.

> #. Edit or create a new record.
> #. Select a name for distinguishing that type.
> #. Select the invoice status for settling the commissions:

Reviewer: ```suggestion
#. Edit or create a new record to select the invoice status for settling the commissions.
```
To keep it more concise.
